omniledger/service/service.go

```diff
@@ -28,6 +27,9 @@ const invokeEvolve darc.Action = darc.Action("invoke:evolve")
 
 // OmniledgerID can be used to refer to this service
 var OmniledgerID onet.ServiceID
+
+const collectTxProtocol = "CollectTxProtocol"
```

Comment:
that should be in `collect_tx.go`

Reply:
I put it here because `CollectTx` is not a "full" protocol, it needs to be registered with a `getTxs` callback. This is similar to how byzcoinx does it.